File: Curso Logica de Programação Neri Neitzke/Algoritmos em Pascal/pascal/crescenteAB.pas
// Prof Neri Aldoir Neitzke - Universidade Ulbra
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
{Problema: Crie 2 variáveis (A e B) e leia 1 valor para cada um deles.
Disponibilize eles de forma crescente.}

program crescenteAB;
    var A, B : integer;
begin
    //A := 11;
    //B := 11;
    write('Digite o valor para A.: ');
    readln(A);
    write('Digite o valor para B.: ');
    readln(B);


    if A > B then
          writeln('Em ordem crescente B=', B ,' - A=',A)
    else
          writeln('Em ordem crescente A=', A ,' - B=',B);

   writeln('..................Final do Sistema..................... ');
   readln();
end.

File: Curso Logica de Programação Neri Neitzke/Algoritmos em Java/crescenteAB.java
// Prof Neri Aldoir Neitzke
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
/*Problema: Crie 2 variáveis (A e B) e leia 1 valor para cada um deles.
Disponibilize eles de forma crescente.
*/
import javax.swing.JOptionPane;
class crescenteAB
{
   public static void main(String args[])
   {
       int a,b;
       a = Integer.parseInt(JOptionPane.showInputDialog("digite o valor de A"));
       b = Integer.parseInt(JOptionPane.showInputDialog("digite o valor de B"));
       if (a > b)
          JOptionPane.showMessageDialog(null,"Em ordem crescente B="+b+" - A="+a);
       else
          JOptionPane.showMessageDialog(null,"Em ordem crescente A="+a+" - B="+b);
   }
}